src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/DetailsPartials/_ChangesToThisApprenticeshipBanner.cshtml

```diff
@@ -0,0 +1,37 @@
+﻿@inject ILinkGenerator LinkGenerator
+@using SFA.DAS.CommitmentsV2.Types
+@using SFA.DAS.EmployerCommitmentsV2.Web.Extensions
+@using SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice
+@model SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice.ApprenticeshipDetailsRequestViewModel
+
+@{
+    var changeToApprenticeshipBanners = Model.GetChangeToApprenticeshipBanners();
+}
+
+@if (changeToApprenticeshipBanners > ChangeToApprenticeshipBanner.None)
+{
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-4">Changes to this apprenticeship</h2>
+
+    @if (changeToApprenticeshipBanners.HasFlag(ChangeToApprenticeshipBanner.PendingChangeWaitingForApproval))
```

Comment:
`HasFlag()` check could move 1 level down, inside the `<div>`

Reply:
That would produce an empty 'div' tag when the check was not met